.nsprc

```diff
@@ -13,6 +13,9 @@
     "https://nodesecurity.io/advisories/65",
 
     // negotiator
-    "https://nodesecurity.io/advisories/106"
+    "https://nodesecurity.io/advisories/106",
+
+    // minimatch, waiting for PR's to land
```

Comment:
I'd probably just go for minimalism and say `// minimatch` so future travellers don't have to try and understand exactly what PR's we're waiting on in which modules.